mmv1/templates/terraform/examples/ces_app_basic.tf.tmpl

```diff
@@ -1,3 +1,55 @@
+data "google_project" "project" {}
+
+resource "google_secret_manager_secret" "fake_private_key_secret" {
+  secret_id = "fake-pk-secret-app-tf"
+
+  replication {
+    auto{}
+  }
+}
+
+resource "google_secret_manager_secret_version" "fake_secret_version" {
+  secret = google_secret_manager_secret.fake_private_key_secret.id
+
+  # copied from mmv1/third_party/terraform/services/compute/test-fixtures/test.key
+  secret_data = <<-EOT
```

Comment:
You should be able to replace the entire key with: `secret_data = file("test-fixtures/test.key")` to clean this up a bit.